source/sync/migrate-sync-modes.txt

```diff
@@ -0,0 +1,376 @@
+
+- The injected field follows this structure: ``<yourPartitionKey> == 
+  <yourPartitionValue>``. The field key is derived from your old
+  Partition-Based Sync  **backend**  configuration. The value is derived from
+  your current Partition-Based Sync  **client**  configuration.
+- The injected field does not sync back to the client that created the object.
+  This means the originating client won't be able to read the injected
+  ``partitionKey`` field.
+
+The Partition-Based Sync client automatically creates sync subscriptions for 
+each table where ``partitionKey == partitionValue``. This will continue
+until you :ref:`migrate your client app <realm-sync-migrate-client>` to
+Flexible Sync.
+
+After migrating your client app to Flexible Sync, the backend will stop injecting
+the partition key field. If your client app relies on the partition key field for
+managing sync subscriptions, you should add the parition key field to your
+object models.
+
+If you don't, objects created in new versions of your app will not be synced
+to clients using older versions of your client app code.
+
+.. _realm-sync-migrate-client:
+
+Migrate Client App to Flexible Sync
+-----------------------------------
+
+After your App's backend has been migrated to Flexible Sync, we recommend
+that you migrate your client app to Flexible Sync. When you migrate your
+client app to Flexible Sync, the client will stop automatically creating
+sync subscriptions for each Realm object model in your app's data model.
+
+We also recommend removing all sync subscriptions and then creating new subscriptions
+for your data. This is the clearest way to control what data is synced after a
+migration. 
+
+If you want to remove individual subscriptions, the automatically
+generated subscriptions use a specific naming format: ``flx_migrated_`` + your 
+object model's name. For example, a ``Person`` object model's subscription
+name would be ``flx_migrated_Person``.
+
+Refer to the SDK documentation for details about adding and removing
+subscriptions:
+
+- Java SDK
+   :ref:`Flexible Sync - Java SDK <java-flexible-sync>`
+
+- Kotlin SDK
+   :ref:`Subscribe to Queryable Fields - Kotlin SDK <kotlin-subscriptions>`
+
+- .NET SDK
+   :ref:`Manage Flexible Sync Subscriptions - .NET SDK <dotnet-flexible-sync>`
+
+- Node.js SDK
+   :ref:`Flexible Sync - Node.js SDK <node-flexible-sync>`
+
+- React Native SDK
+   :ref:`Flexible Sync - React Native SDK <react-native-flexible-sync>`
+
+- Swift SDK
+   :ref:`Manage Flexible Sync Subscriptions - Swift SDK <swift-manage-flexible-sync-subscriptions>`
+
+You can keep the automatically-generated subscriptions if you don't want to
+remove and recreate them. But you should consider that because these 
+subscriptions aren't created in your client code, they may make it harder to
+maintain and extend client code. 
+
+Automatically-generated subscriptions will not exist on clients that never
+connect to your App after it has been migrated to Flexible Sync.
```

Comment:
Can't remember the context around why this note was added, but it's unclear why it's important if the client is not re-connecting? Maybe it's referring to a client app that connects as FLX after the migration without ever having connected as PBS? Probably worth clarifying if that's the case

Reply:
Agreed, it's not clear. I think we could safely remove, as this should be self-evident based on all the behavior described elsewhere on the page.